lldb/source/Core/CoreProperties.td

```diff
@@ -172,6 +172,14 @@ let Definition = "debugger" in {
     Global,
     DefaultStringValue<"${ansi.normal}">,
     Desc<"When displaying progress in a color-enabled terminal, use the ANSI terminal code specified in this format immediately after the progress message.">;
+  def ShowStatusline: Property<"show-statusline", "Boolean">,
+    Global,
+    DefaultTrue,
+    Desc<"Whether to show a statusline at the bottom of the terminal.">;
+  def StatuslineFormat: Property<"statusline-format", "FormatEntity">,
+    Global,
+    DefaultStringValue<"${ansi.bg.blue}${ansi.fg.black}{${target.file.basename}}{ | ${line.file.basename}:${line.number}:${line.column}}{ | ${thread.stop-reason}}{ | {${progress.count} }${progress.message}}">,
+    Desc<"The default statusline format string.">;
```

Comment:
"the default" implies that there is some other setting to change the format string, when in reality this is the one you change. It has a default value though of course.

Reply:
Also can we use words that if I search the lldb website, return something useful? I think "format string" would work.

Reply:
This mimics the help text of the other format strings, which all use "The default <> format string". I'm open to writing something better (for all of them). I'm open to suggestions but my ideas were either repetitive ("The <> format string, which can be customized using LLDB's format strings") or would require referring to the website or docs, which we don't do anywhere else ("The <> format string. See the LLDB website/documentation to see what format strings are available"). LLDB is pretty self contained so I don't really like the latter, even though it's objectively the better option.

Reply:
Yes that would be redundant and it doesn't answer the question of what a format string actually is and how I enumerate them. I can infer some of it from the default but I'm limited to the ones used there, which is a shame if I wanted to do something advanced with it.

Ideally we'd have some help that didn't need to be tied to a command, like a topic page. Then these strings can refer to `help <topic>`, assuming there's some way to resolve the clash with command names there. Or we choose one command to get a full explanation and refer to that, but this is also a bit of a hack.

It's fine as is for a first cut but mostly because people are used to software being vague, but we can do better I'm sure.

Reply:
I've filed https://github.com/llvm/llvm-project/issues/130997